File: python/pad_array.py
#REMEMBER TO PSEUDOCODE

# find length of Array, if length is less than minsize, push the filler word (while loop)
def pad(array, min_size, value = None):
    answer=array.copy()
    while len(answer) < min_size:
        answer.append(value)
    return answer

print(pad([1,2,3], 5))
print(pad([1,2,3], 5, 'apple'))
print(pad([1,2,3], 3, 'apple'))

File: js/padArray.js
// REMEMBER TO PSEUDOCODE
// const pad = (array, minSize, value=null) => {

// }

// find length of Array, if length is less than minsize, push the filler word (while loop)

function pad(array, minSize, value=null) {
    let answer =[...array]
    while (answer.length < minSize) {
        answer.push(value)
    }
    return answer
}
console.log(pad([1,2,3], 5))
console.log(pad([1,2,3], 5, 'apple'))